<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditableQuestionDialog</class>
 <widget class="QDialog" name="EditableQuestionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>535</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="questionNumberHorizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>number: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="questionNumberLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="questionNumberHorizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="questionHorizontalLayout">
     <item>
      <widget class="QLabel" name="questionLabel">
       <property name="text">
        <string>question:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="questionTextEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="answersVerticalLayout">
     <item>
      <widget class="QLabel" name="answersLabel">
       <property name="text">
        <string>answers:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="answersHorizontalLayout">
       <item>
        <widget class="QListView" name="listView"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="answersButtonsVerticalLayout">
         <item>
          <widget class="QPushButton" name="addRowButton">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteRowButton">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="descriptionHorizontalLayout">
     <item>
      <widget class="QLabel" name="descriptionLabel">
       <property name="text">
        <string>description:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="descriptionTextEdit">
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:7.8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="sourceHorizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>source:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="sourceTextEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="tagsHorizontalLayout">
     <item>
      <widget class="QLabel" name="tagsLabel">
       <property name="text">
        <string>tags:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="tagsPushButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditableQuestionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>523</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditableQuestionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>523</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteRowButton</sender>
   <signal>clicked()</signal>
   <receiver>EditableQuestionDialog</receiver>
   <slot>removeAnswersListRows()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>624</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>552</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteRowButton</sender>
   <signal>clicked()</signal>
   <receiver>listView</receiver>
   <slot>doItemsLayout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>624</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addRowButton</sender>
   <signal>clicked()</signal>
   <receiver>EditableQuestionDialog</receiver>
   <slot>addAnswersListRows()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>624</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>638</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addRowButton</sender>
   <signal>clicked()</signal>
   <receiver>listView</receiver>
   <slot>doItemsLayout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>624</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addQuestionToDb()</slot>
  <slot>removeAnswersListRows()</slot>
  <slot>addAnswersListRows()</slot>
 </slots>
</ui>
